### PR TITLE
Spelling core UI (non code)

### PR DESCRIPTION
**Pull request by jsoref**
This is split from #35947

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: static/app/components/forms/controls/rangeSlider/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {forwardRef as reactFowardRef, useEffect, useState} from 'react';
+import {forwardRef as reactForwardRef, useEffect, useState} from 'react';
```
Comment (priscilawebdev) [inline static/app/components/forms/controls/rangeSlider/index.tsx:1]: in this case, we will have to update this line too 
https://github.com/getsentry/sentry/blob/9a6320baa48450d9e7fec4a530fe9cfe3110a43e/static/app/components/forms/controls/rangeSlider/index.tsx#L209

**Comment by priscilawebdev** [comment]
Thanks for contributing @jsoref 🙏👏 I've left a comment but other than that it looks good to me

File: static/app/components/forms/controls/rangeSlider/index.tsx (hunk 1 above)
Comment (jsoref) [reply static/app/components/forms/controls/rangeSlider/index.tsx:1]: Oh. This project seems to have a whitespace rule which is forcing VSCode to rewrite the entire file when I choose `save`. So I tried "manually" fixing the thing up, which is how I (I'm human, this is the proof!) messed up.

**Comment by evanpurkhiser** [comment]
Looks good. I do kind of wish that one little code change wasn't in there (Fixing the spelling on `reactFowardRef`) but other than that all great fixes.

Thanks so much for this @jsoref !!

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**